runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -122,6 +122,50 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- A profile that adds an integration test phase for streaming if and only if
+         the streamingValidatesRunnerPipelineOptions maven property has been set.
+         It should be set to a valid PipelineOptions JSON string. -->
+    <profile>
```

Comment:
why we need a profile?

i think the whole story would be cleaner if we just had two executions, no profiles.